PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+# Description
```

Comment:
Prefer putting this file either in .github or docs directory. That is the standard convention.